app.js

```diff
@@ -1 +1,235 @@
+//     }
+// }
+// let newCar = new Car('Lamborghini', 2, true, 'ZedBlack', 'Petrol', 6498);
+// newCar.make();
+// // Arithmetic operation on two numbers
+// // addition
+// function addNumbers(a: number, b: number) { 
+//     return a + b; 
+// } 
+// var sum: number = addNumbers(10, 15) 
+// console.log('Sum of the two numbers is: ' +sum); 
+// // Subtraction
+// function subtractNumbers(a: number, b: number) { 
+//     return a - b; 
+// } 
+// var Difference: number = subtractNumbers(10, 15) 
+// console.log('Difference of the two numbers is: ' +Difference); 
+// // Multiplication
+// function multiplyNumbers(a: number, b: number) { 
+//     return a * b; 
+// } 
+// var sum: number = multiplyNumbers(10, 15) 
+// console.log('Multiplication of the two numbers is: ' +sum); 
+// // Divison
+// function divideNumbers(a: number, b: number) { 
+//     return a / b; 
+// } 
+// var sum: number = divideNumbers(10, 15) 
+// console.log('Division of the two numbers is: ' +sum); 
+// // Type Annotations in TypeScript
+// var age: number = 32; // number variable
+// var tName: string = "John";// string variable
+// var isUpdated: boolean = true;// Boolean variable
+// function display(id:number, tName:string)
+// {
+//     console.log(`My name is ${tName}. My Age is ${id}`)
+// }
+// display(32, "John");
+// letDeclaration();
+// let num1:number = 1; 
+// function letDeclaration() { 
+//     let num2:number = 2; 
+//     if (num2 > num1) { 
+//         let num3: number = 3;
+//         num3++; 
+//     } 
+//     while(num1 < num2) { 
+//         let num4: number = 4;
+//         num1++;
+//     }
+//     console.log(num1); //OK
+//     console.log(num2); //OK 
+//     // console.log(num3); //Compiler Error: Cannot find name 'num3'
+//     // console.log(num4); //Compiler Error: Cannot find name 'num4'
+// }
+// var declaration
+// var numb1:number = 1; 
+// function varDeclaration() { 
+//     var num2:number = 2; 
+//     var num3: number;
+//     if (num2 > numb1) { 
+//         num3 = 3;
+//         num3++; 
+//     } 
+//     while(numb1 < num2) { 
+//         var num4: number = 4;
+//         numb1++;
+//     }
+//     console.log(numb1); //2
+//     console.log(num2); //2 
+//     console.log(num3); //4
+//     console.log(num4); //4
+// }
+// varDeclaration();
+// let str: string = 'TypeScript';
+// str.indexOf('T'); // returns 0
+// str.indexOf('p'); // returns 2
+// str.indexOf('e'); // returns 3
+// str.indexOf('T', 1); // returns -1
+// str.indexOf('t', 1); // returns 9
+// let fruits: string[] = ['Apple', 'Orange', 'Banana']; 
+// for(var index in fruits)
+// { 
+//     console.log(fruits[index]);  // output: Apple Orange Banana
+// }
+// for(var i = 0; i < fruits.length; i++)
+// { 
+//     console.log(fruits[i]); // output: Apple Orange Banana
+// }
```

Comment:
Dude. Why this whole mess of comment and uncomment code? Just remove the code. You can always go back in git history and recover any code you want. 

Please understand the value of code readability. It should not look like a Garbage dump.